module/VuFind/src/VuFind/Db/Entity/Search.php

```diff
@@ -127,6 +131,13 @@ class Search implements SearchEntityInterface
      */
     protected $searchObject;
 
+    /**
+     * Normalized search object after loading.
+     *
+     * @var \VuFind\Search\Minified|null
```

Comment:
More concise syntax:

```suggestion
     * @var ?\VuFind\Search\Minified
```